### PR TITLE
IGVF-1756-CRISPR-NTRs-audit

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/audit/file_set.py [hunk 1]
```diff
@@ -34,7 +34,15 @@ def audit_no_files(value, system):
...
-    if not (value.get('files', '')):
+    assay_term = value.get('assay_term', '')
+    if assay_term:
+        assay_term = system.get('request').embed(assay_term, '@@object?skip_calculated=true')
+        assay_term = assay_term.get('term_name', '')
+    # Measurement sets from these CRISPR assays do not expect any sequence data
+    CRISPR_assays = ['proliferation CRISPR screen',
+                     'CRISPR perturbation screen followed by flow cytometry and FISH']
+    preferred_assay_title = value.get('preferred_assay_title', '')
+    if not (value.get('files', '')) and assay_term not in CRISPR_assays and preferred_assay_title not in ['Variant FlowFISH']:
```
Comment (gabdank) [inline src/igvfd/audit/file_set.py:45]: should not we rely on Cas mediated mutagenesis instead? https://data.igvf.org/assay-terms/OBI_0003133/ why variant flowfish is the only one relying on preferred title?

Comment (gabdank) [reply src/igvfd/audit/file_set.py:45]: exactly as it appears in measurement set.py below

File: src/igvfd/audit/measurement_set.py [hunk 2]
```diff
@@ -211,24 +212,43 @@ def audit_missing_institutional_certification(value, system):
...
         {
             "audit_description": "Measurement sets from CRISPR-based assays are expected to link to a gRNA sequencing auxiliary set.",
             "audit_category": "missing auxiliary set",
             "audit_level": "NOT_COMPLIANT"
-        },
+        }
+    ]
+    '''
+    description = get_audit_description(audit_CRISPR_screen_missing_auxiliary_set)
+    if value['assay_term']['term_name'] in ['proliferation CRISPR screen',
```
Comment (gabdank) [inline src/igvfd/audit/measurement_set.py:226]: should not we include variant flowfish here?

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/audit/file_set.py (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/audit/file_set.py:45]: I wasn't sure if other cas mediated mutagenesis screens would not expect to have files in the measurement set.

File: src/igvfd/audit/measurement_set.py (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/audit/measurement_set.py:226]: Variant FlowFISH doesn't have gRNA sequencing

**Review by gabdank — APPROVED** [review]

**Pull request merged**